frontend/src/components/routerpages/Search.js

```diff
@@ -2,23 +2,31 @@ import React from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
 
-import Header from './Header';
-import ProductList from './ProductList';
-import QueryComponents from './QueryComponents';
-import { deviceSize } from '../theme';
-import CategoryContainer from '../containers/categoryContainer';
-import FeaturesContainer from '../containers/featuresContainer';
-import SubcategoriesContainer from '../containers/subcategoriesContainer';
-import FeatureList from './FeatureList';
+import Header from '../Header';
+import ProductList from '../ProductList';
+import QueryComponents from '../QueryComponents';
+import { deviceSize } from '../../theme';
+import CategoryContainer from '../../containers/categoryContainer';
+import FeaturesContainer from '../../containers/featuresContainer';
+import SubcategoriesContainer from '../../containers/subcategoriesContainer';
+import FeatureList from '../FeatureList';
 
-const Wrapper = styled.div`
+const HeaderWrapper = styled.div`
```

Comment:
I don't think you need this since the width of the parent component is already at `100%`

Reply:
done